src/linearmodel/glm.jl

```diff
@@ -45,7 +43,7 @@ function gethistory(
 end
 
 function sethistory!(
```

Comment:
set history -> set_history!

Reply:
Good call, I'll go through and do sethistory! -> set_history! and gethistory -> get_history.